js/impact.js

```diff
@@ -0,0 +1,2332 @@
+               break;
+            }
+            $(impact.impactContainer).css('cursor', "grab");
+            break;
+
+         case EDITION_ADD_NODE:
+            break;
+
+         case EDITION_ADD_EDGE:
+            break;
+
+         case EDITION_DELETE:
+            if (event.target.data('id') == undefined) {
+               break;
+            }
+
+            $(impact.impactContainer).css('cursor', "default");
+            var id = event.target.data('id');
+
+            // Remove red overlay
+            event.cy.filter().data('todelete', 0);
+            event.cy.filter().unselect();
+
+            // Store here if one default node
+            if (event.target.data('id') == impact.startNode) {
+               $(impact.impactContainer).css('cursor', "not-allowed");
+               break;
+            }
+
+            // Add red overlay
+            event.target.data('todelete', 1);
+            event.target.select();
+
+            if (event.target.isNode()){
+               var sourceFilter = "edge[source='" + id + "']";
+               var targetFilter = "edge[target='" + id + "']";
+               event.cy.filter(sourceFilter + ", " + targetFilter)
+                  .data('todelete', 1)
+                  .select();
+            }
+            break;
+      }
+   },
+
+   /**
+    * Handle global mouseout events
+    *
+    * @param {JQuery.Event} event
+    */
+   onMouseout: function(event) {
+      switch (impact.editionMode) {
+         case EDITION_DEFAULT:
+            $(impact.impactContainer).css('cursor', "move");
+            break;
+
+         case EDITION_ADD_NODE:
+            break;
+
+         case EDITION_ADD_EDGE:
+            break;
+
+         case EDITION_DELETE:
+            // Remove red overlay
+            $(impact.impactContainer).css('cursor', "move");
+            event.cy.filter().data('todelete', 0);
+            event.cy.filter().unselect();
+            break;
+      }
+   },
+
+   /**
+    * Handle "goTo" menu event
+    *
+    * @param {JQuery.Event} event
+    */
+   menuOnGoTo: function(event) {
+      window.open(event.target.data('link'), 'blank');
+   },
+
+   /**
+    * Handle "showOngoing" menu event
+    *
+    * @param {JQuery.Event} event
+    */
+   menuOnShowOngoing: function(event) {
+      $(impact.dialogs.ongoingDialog.id).html(
+         impact.buildOngoingDialogContent(event.target.data('ITILObjects'))
+      );
+      $(impact.dialogs.ongoingDialog.id).dialog(impact.getOngoingDialog());
+   },
+
+   /**
+    * Handle "EditCompound" menu event
+    *
+    * @param {JQuery.Event} event
+    */
+   menuOnEditCompound: function (event) {
+      $(impact.dialogs.editCompoundDialog.id).dialog(
+         impact.getEditCompoundDialog(event.target)
+      );
+   },
+
+   /**
+    * Handler for "removeFromCompound" action
+    *
+    * @param {JQuery.Event} event
+    */
+   menuOnRemoveFromCompound: function(event) {
+      var parent = impac
```

Comment:
I have a JS error when I click on the export link:
![image](https://user-images.githubusercontent.com/33253653/64534520-e3bb7d00-d315-11e9-9dbc-177deade35ba.png)